lms/djangoapps/edx_solutions_api_integration/courses/serializers.py

```diff
@@ -11,7 +11,7 @@ class GradeSerializer(serializers.Serializer):
 
 class CourseLeadersSerializer(serializers.Serializer):
     """ Serializer for course leaderboard """
-    id = serializers.IntegerField(source='user__id')
+    id = serializers.IntegerField(source='user__id')  # pylint: disable=C0103
```

Comment:
instead of using pylint numerical directives we should use named directives for readability i.e. `# pylint: disable=invalid-name`